cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -42,6 +42,14 @@ function(handle_gyb_source_single dependency_out_var_name)
       ${GYB_SINGLE_FLAGS})
 
   set(gyb_tool "${SWIFT_SOURCE_DIR}/utils/gyb")
+
+  # GYB needs to be run via python on Windows
+  if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    set(gyb_tool_command "python" ${gyb_tool})
+  else()
+    set(gyb_tool_command ${gyb_tool})
+  endif()
+
```

Comment:
You can always run it this way irrespective of the platform.  I think having a single way to run it is better.